es/outline/intro.md

```diff
@@ -31,71 +30,81 @@ Introducción a la Programación en Clojure
 
-> Clojure es de _proposito general_. Algunos lenguajes tienen un foco específico.
-> JavaScript, por ejemplo, fue tradicionalmente usado solo en páginas web
-> (aunque eso ha cambiado de alguna manera). Objective-C es usado principalmente
-> para aplicaciones iPhone. Hoy vamos a hacer una aplicación para dibujar,
-> pero puedes usar Clojure para cualquier tipo de aplicación facilmente.
+> Clojure es _de propósito general_. Algunos lenguajes están focalizados
+> en algo específico.
+> JavaScript, por ejemplo, en un principio fue usado sólo en páginas web
+> (aunque eso ha cambiado de alguna manera). Objective-C se usa principalmente
+> para aplicaciones iPhone.
+> Si bien hoy vamos a hacer una aplicación para dibujar,
+> fácilmente podés usar Clojure para cualquier tipo de aplicación.
 {: ng-show="block13" .description}
 
-#### Clojure es _diversión_ <button class="link" ng-bind-html="details" ng-model="block14" ng-click="block14=!block14"></button>
+#### Clojure es _divertido_ <button class="link" ng-bind-html="details" ng-model="block14" ng-click="block14=!block14"></button>
 
-> Clojure es _diversión_. Eso es una cuestión de opinión, por supuesto, pero
-> pensamos que es verdad. Esperamos que durante este curso experimenten
-> la alegría de construir un programa hecho en Clojure y hacer algo
-> poderoso y sorprendente.
+> Clojure es _divertido_. Si bien eso es una cuestión de opinión, nosotros pensamos
+> que es así. En este curso, esperamos que, mientras construís un programa en
+> Clojure, experimentés la alegría de crear algo poderoso y sorprendente.
 {: ng-show="block14" .description}
 </section>
 
 <section ng-controller="NarrativeController">
-## ¿En qué es bueno Clojure?
+## ¿Para qué es bueno Clojure?
 {: .slide_title .slide}
 
 #### <button class="link" ng-model="block21" ng-click="block21=!block21">Intro</button>
 
-> Entonces, dijimos que Clojure es de propósito general, y lo es. Eso no significa
-> que no tenga puntos fuertes.
+> Entonces, dijimos que Clojure es de propósito general, y lo es. Claro que tiene
+> sus puntos fuertes.
 {: ng-show="block21" .description}
 
 #### Procesamiento de datos <button class="link" ng-bind-html="details" ng-model="block22" ng-click="block22=!block22"></button>
 
-> Clojure es reconocido por hacer un buen procesamiento de datos. Esto es
-> porque tiene un buen set de estructuras de datos. Es decir, que tiene muchas
-> maneras incorporadas de representar datos que lo hacen fácil y poderoso.
+> Clojure es reconocido por ser bueno para procesar datos. Esto es
+> porque tiene un buen conjunto de estructuras de datos. Es decir, que ya tiene
+> incorporadas muchas maneras de representar datos útiles y poderosas.
 {: ng-show="block22" .description}
 
 #### Concurrencia <button class="link" ng-bind-html="details" ng-model="block23" ng-click="block23=!block23"></button>
 
-> Clojure es reconocido por su concurrencia. Pensá acerca de escribir
```

Comment:
This part ended up too long. However, it is the only way I have found so far to convey the concepts...